charts/evm-rollup/Chart.yaml

```
@@ -15,7 +15,7 @@ type: application
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
 # Versions are expected to follow Semantic Versioning (https://semver.org/)
-version: 0.25.0
+version: 0.26.0
```

```suggestion
version: 0.25.1
```

we can call this one a patch since the last bump was effectively running same code